### DISCUSSION TITLE
Captured image does not correspond to live stream or reference image

### DISCUSSION THREAD

**Discussion by osmeest** [jomjol/AI-on-the-edge-device#2689] [General] [answered]
My ESP32CAM is not straight in front of the electricity counter. In the initial setup, I had to rotate the image a bit.
After working ok for a couple of days, it suddenly stopped recognizing. The captured image was slightly moved.
So I reconfigured (new reference image, new markers, reposition all digits). In the new configuration there is no rotation anymore but the captured image does not correspond to the reference image and so, digits are not properly recognised anymore.

Captured image in overview:
<img width="1042" alt="captured_image" src="https://github.com/jomjol/AI-on-the-edge-device-docs/assets/355037/c015aed2-70d2-4faf-899a-cc5dfb51cf24">
Live stream:
<img width="641" alt="live_stream" src="https://github.com/jomjol/AI-on-the-edge-device-docs/assets/355037/90f696bf-aeee-47a3-b7c1-3af9c4d38777">
Reference image (marker config):
<img width="673" alt="reference_image" src="https://github.com/jomjol/AI-on-the-edge-device-docs/assets/355037/cd221139-bbaa-4551-8c53-8e4665477561">

How can I fix this ?
Is there a way to restart the setup process ?

Thanks !

**Comment by caco3** [comment, marked as answer]
- The reference Marks should be as far apart as possible. Also the left mark should be made smaller.
- Since the whole view is inclined, the ROIs on the right might need to be smaller than the ones on the left side. If you could move the camera to the right that surly would make this easier.
- There is no need to restart the setup mode, you can access all setting steps through the settings menu (ref image, ref marks, ROIs, config)

**Reply by osmeest** [reply]
Thanks for the good advices.
I updated my physical setup to be more in front of the counter.
![updated_setup](https://github.com/jomjol/AI-on-the-edge-device/assets/355037/e36382c9-4856-437e-b0a4-6ed4a28078dc)

Thanks to that, I could get a better picture (without the trapezoid perspective).
Also moved the alignment markers as advised.
<img width="1027" alt="updated_config" src="https://github.com/jomjol/AI-on-the-edge-device/assets/355037/0344ccf2-c685-4ca9-90c5-bab65ae681d5">

Let's hope it stays well aligned from now on ;)

Thanks for the kind support.

Regards,

**Comment by friedpa** [comment]
Put a black tape or sticker on the centre of the reflection to invrease the contrast. Or use a trick mentioned in #1949 (Pol Filter Foil)
